lib/parse-args.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const minimist = require('minimist');
+
+const parse = (args) => {
+  if (!args || args.length === 0) {
+    throw new Error('No src folder or pattern specified');
+  }
+  const parsed = minimist(args);
+  const writeShared = parsed.writeShared || parsed.w;
+
+  let sources = parsed._;
+  let shared = parsed.s || parsed.shared;
+
+  if (shared && !Array.isArray(shared)) {
+    shared = [shared];
+  }
```

Comment:
Yeah but if it's already an array then the call will be `[1,2,3].concat([1, 2, 3]) => [ 1, 2, 3, 1, 2, 3 ]`, unless I'm missing something, in which case, I will fall back to saying: another PR